specification/web/resource-manager/Microsoft.Web/stable/2020-06-01/ResourceProvider.json

```diff
@@ -15,6 +15,41 @@
     "application/json"
   ],
   "paths": {
+    "/providers/Microsoft.Web/generateGithubAccessTokenForAppserviceCLI": {
```

Comment:
It is not allowed to add a new api into an existing api-version.  Please see the breaking change document (and linked API guidelines) from the checklist in PR comments.

Reply:
Updated to new version, thanks